tests/jax/tpu-embedding.libsonnet

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+local common = import 'common.libsonnet';
+local mixins = import 'templates/mixins.libsonnet';
+local tpus = import 'templates/tpus.libsonnet';
+
+{
+  local tpu_embedding = self.tpu_embedding,
+  tpu_embedding:: common.JaxTest {
+    local config = self,
+    frameworkPrefix: 'jax.tpu.embedding',
```

Comment:
It's unit test, so I'd suggest we keep the similar pattern with what we have. Do you mind renaming it something like `jax-tpu-embedding-pjrt-func-v4-8`? So we have consistent name shown in the dashboard.